graphql2/graphqlapp/slack.go

```diff
@@ -116,6 +112,7 @@ func (q *Query) GenerateSlackAppManifest(ctx context.Context) (string, error) {
 	}
 	var t bytes.Buffer
 	cfg := config.FromContext(ctx)
+	// sets ApplicationName and CallbackURL from config
```

Comment:
I think this can be left out, nothing is being set here, we're just passing config as the template context
```suggestion
```